webargs/core.py

```diff
@@ -245,9 +248,13 @@ def _validate(self, name, value):
                     val = ret[key]
                 except KeyError:
                     if nested_arg.required:
-                        raise RequiredArgMissingError(
-                            'Required parameter "{0}" not found.'.format(key)
-                        )
+                        req_type = (nested_arg.required).__class__.__name__
+                        if req_type in ['str', 'unicode']:
```

Comment:
Why not `if isinstance(nested_arg.required, basestring)` (you would have to add `basestring = (str, bytes)` up top for py3 compat)?